### PR TITLE
Allow lower-case words until ignoring <code> tag works

### PR DESCRIPTION
**Pull request by MichaIng**
to work around failing <code> tag exclusion: https://github.com/MichaIng/DietPi-Docs/pull/96

Little test, if I didn't see if wrong, "WiFi" is already part of the default dictionary.

### CODE REVIEW

File: .markdownlintrc [hunk 1]
```diff
@@ -4,7 +4,9 @@
     "style": "atx"
   },
   "MD013": false,
-  "MD024": false,
+  "MD024": {
+    "allow_different_nesting": true,
```
Comment (StephanStS) [inline .markdownlintrc:8]: I assume the trailing ',' is wrong in this line 8 after 'true'.

-> "MD024": { "allow_different_nesting": true};

**Review by StephanStS — COMMENTED** [review]
Beside the 'wifi' the spell check also needs some more words (in the shown small letter variant):
- justboom
- letsencrypt
- nordvpn
- cpuinfo
- bugreport
- morsecode
- logclear

See section "Check spelling in HTML files" there: <https://github.com/MichaIng/DietPi-Docs/runs/1271999878?check_suite_focus=true>.

File: .markdownlintrc (hunk 1 above)
Comment (MichaIng) [reply .markdownlintrc:8]: Oh yes it is, thanks for the hint!

**Comment by MichaIng** [comment]
Ah yes now that the capital headers have been changed to lower-case code. Let's add them for now until there is a real solution.

**Pull request merged**